input/pages/codings.xml

```diff
@@ -102,15 +102,15 @@
     <td>CLINVAR-V</td>
   </tr>
   <tr>
-    <td>ClinVar Accesion ID (RCV)<br/>(NCBI/NLM)</td>
+    <td>ClinVar Accession ID (RCV)<br/>(NCBI/NLM)</td>
     <td>RCV' refers to the first 3 letters of the accession calculated by ClinVar to aggregate information from all submissions interpreting the same phenotype relative to the same variant or set of variants. </td>
     <td>https://www.ncbi.nlm.nih.gov/clinvar<br/>(not assigned yet)</td>
     <td>not assigned yet</td>
     <td></td>
   </tr>
   <tr>
     <td>COSMIC<br/>(Wellcome Trust Sanger Institute)</td>
-    <td>This table includes only simple somatic (cancer) mutations, one per unique mutation ID. The code is the COSMIC mutation ID, and the name is constructed from Ensembl transcript reference sequences and p.HGVS that use the single letter codes for amino acids. It carries fields analogous to most of the key fields in ClinVar, but its referencesequences are Ensembl transcript reference sequences with prefixes of ENST; it specifies amino acid changeswith the older HGVS single letter codes and it carries examples of primary cancers and primary tissues - fields thatare not in ClinVar.COSMIC's source table includes multiple records per mutation - one per submission. The COSMIC- SimpleVariants table that we have extracted from the original file includes only one record per unique mutation – a total ofmore than 3 million records.These contents are copyright COSMIC (http://cancer.sanger.ac.uk/cosmic/license). LHC has produced a look uptable for these records, and for users to look up particular mutation IDs, both with permission from COSMIC. However, interested parties must contact COSMIC directly for permission to download these records.</td>
+    <td>This table includes only simple somatic (cancer) mutations, one per unique mutation ID. The code is the COSMIC mutation ID, and the name is constructed from Ensembl transcript reference sequences and p.HGVS that use the single letter codes for amino acids. It carries fields analogous to most of the key fields in ClinVar, but its reference sequences are Ensembl transcript reference sequences with prefixes of ENST; it specifies amino acid changes with the older HGVS single letter codes and it carries examples of primary cancers and primary tissues - fields that are not in ClinVar.COSMIC's source table includes multiple records per mutation - one per submission. The COSMIC- SimpleVariants table that we have extracted from the original file includes only one record per unique mutation – a total of more than 3 million records. These contents are copyright COSMIC (http://cancer.sanger.ac.uk/cosmic/license). LHC has produced a lookup table for these records, and for users to look up particular mutation IDs, both with permission from COSMIC. However, interested parties must contact COSMIC directly for permission to download these records.</td>
```

Comment:
Looks like we need a space:

`ClinVar.COSMIC's`

Reply:
I added it.